packages/react/src/components/Table/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export {default} from './Table';
+export * from './Table';
```

Comment:
Export the public API from `@mui/material/Table` from here as well.

1. `ctrl` + click on an  `@mui/material/Table` import.
2. click on `export * from './Table';` in the `index.d.ts`
2. And then check what are the ones we can export.

ex: `Table`, `TableProps` are already exported by our override but we can export additional ones like the following.


<img width="919" alt="Screenshot 2025-01-09 at 14 18 50" src="https://github.com/user-attachments/assets/881bd820-6b7a-41a6-8c1b-fc8ccd12dbc9" />

Reply:
Ref: https://github.com/wso2/oxygen-ui/blob/main/packages/react/src/components/Box/index.ts#L22

Reply:
Fixed with bd50b0399fca298ee28eb3ed1d28e67de1e83052